maskit/masked_parameters.py

```diff
@@ -54,15 +60,22 @@ def reset(self):
         """
         self._mask = np.zeros_like(self._params, dtype=bool, requires_grad=False)
 
-    def copy(self) -> 'MaskedParameters':
+    def copy(self) -> "MaskedParameters":
         clone = object.__new__(MaskedParameters)
         clone._params = self._params.copy()
         clone._mask = self._mask.copy()
         clone.perturbation_axis = self.perturbation_axis
         return clone
 
-    def perturb(self, amount: int = None, mode: PerturbationMode = PerturbationMode.INVERT, random: bool = True):
-        assert amount is None or amount >= 0, "Negative values are not supported, plese use PerturbationMode.REMOVE"
+    def perturb(
+        self,
+        amount: int = None,
```

Comment:
This also needs a ``from typing import Optional`` at the top of the file.
```suggestion
        amount: Optional[int] = None,
```